### DISCUSSION TITLE
Running Ivorysql on docker

### DISCUSSION THREAD

**Discussion by haarse** [IvorySQL/IvorySQL#670] [General]
Hi,

I have just installed ivorysql on Docker.
I want to know what I need to setup to be able to run Oracle sql against the database.

Right now when I run this command:
`CREATE TABLE db_comp_revision (db_type VARCHAR2(80),db_key VARCHAR2(80),db_rev_date DATE,db_revision VARCHAR2(80));`
I receive this error:
ERROR:  type "varchar2" does not exist
LINE 1: CREATE TABLE db_comp_revision (db_type VARCHAR2(80),db_key V...
                                               ^ 
SQL state: 42704
Character: 40

What have I done wrong?

**Comment by balinorLiang** [comment]
The default compatible_mode is pg, just need to run below command, change it, then your command will work.
`ivorysql# SET ivorysql.compatible_mode to oracle;` 
@haarse
